windows/ReactNativePicker/ReactPickerView.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ReactPickerView.g.h"
+#include "winrt/Microsoft.ReactNative.h"
+#include "NativeModules.h"
+
+namespace winrt::ReactNativePicker::implementation {

```

Comment:
>ReactNativePicker [](start = 17, length = 17)

Question:  Does this implementation do the equivalent of supplying a custom yoga measure function like it currently does in react-native-windows?  Picker is special in that it can size itself to the selected text, see PickerViewManager::GetYogaCustomMeasureFunc()

Reply:
@jonthysell (or Keith) what interface or callback should I implement to plug in our own measure function? The one (GetYogaCustomMeasureFunc) we used in react::uwp is an override of the function in [ViewManagerBase](https://github.com/microsoft/react-native-windows/blob/master/vnext/ReactUWP/Views/ViewManagerBase.cpp) which doesn't seem to be accessible here.

I looked at [IViewManager.idl](https://github.com/microsoft/react-native-windows/blob/master/vnext/Microsoft.ReactNative/IViewManager.idl), but didn't find anything related...

Reply:
I don't believe we have any mechanism in place yet to handle this.  It doesn't seem too difficult though in principle, as the code in UWP just needs a XAML FrameworkElement to measure.  

I'm not sure how Picker behaves without this.  Does it require the app to set a width/height?  Ideally it works well enough for us to file a follow-up issue and addressing after this is checked in.

Reply:
Ah I see, I was wondering why it didn't render without width/height, looks like that's the reason. I've added height in the default style so app author doesn't have to set width/height.

https://github.com/react-native-community/react-native-picker/blob/5260f4887137c3417d1d22f7350d8a3fcde7ab2a/js/PickerWindows.windows.js#L137-L140

I'll file a follow-up issue about the measure override.

Reply:
thanks!